voter/utils.py

```diff
@@ -0,0 +1,128 @@
+from datetime import datetime, timezone
+from enum import Enum
+import os
+import subprocess
+
+import requests
+
+from voter.models import FileTracker
+
+
+FETCH_STATUS_CODES = Enum("FETCH_STATUS_CODES",
+                          "CODE_OK CODE_NET_FAILURE CODE_WRITE_FAILURE CODE_NOTHING_TO_DO")
+
+
+def get_output_file(output):
```

Comment:
I like the spirit of this helper and the way you use it in all the `print()` calls but if *all* the `print()` calls now have the same parameters in order to use it, both to `file` and `flush`, maybe a better helper is a simple `out()` or `msg()` that wraps the print call itself? 

I don't feel real strong on it, tho. So take the suggestion at your discretion. Hah! That rhymes!